ROADMAP.md

```diff
@@ -0,0 +1,104 @@
+# `pflag` roadmap
+
+## Beef up the team of maintainers
+
+3 to 5 volunteers. No less, no more.
+
+Would be nice to open a slack or discord channel to make debate more lively.
+Otherwise, we may use github discussions.
```

Comment:
```suggestion
Otherwise, we may use GitHub discussions.
```

Reply:
Both makes sense. We can have a channel on Gophers Slack. GH Discussions is better for archiving.

Reply:
ok let's try both